# Patch to reutrn major VERSION, SOVERSION and LIBTOOL_CURRENT to 12 to avoid
# breaking dependencies, because verstion 2.2.2 is patch only relese including
# only security fixes and no new functionality or API changes
# See https://github.com/openexr/openexr/commit/e69de40ddbb6bd58341618a506b2e913e5ac1797
# See https://github.com/openexr/openexr/issues/250
diff -urNad openexr-2.2.2-orig/IlmBase/CMakeLists.txt openexr-2.2.2/IlmBase/CMakeLists.txt
--- openexr-2.2.2-orig/IlmBase/CMakeLists.txt	2020-04-30 20:45:37.000000000 +0300
+++ openexr-2.2.2/IlmBase/CMakeLists.txt	2020-05-01 14:36:02.621798218 +0300
@@ -5,7 +5,7 @@
 
 SET(ILMBASE_VERSION_MAJOR "2")
 SET(ILMBASE_VERSION_MINOR "2")
-SET(ILMBASE_VERSION_PATCH "1")
+SET(ILMBASE_VERSION_PATCH "2")
 
 SET(ILMBASE_VERSION ${ILMBASE_VERSION_MAJOR}.${ILMBASE_VERSION_MINOR}.${ILMBASE_VERSION_PATCH})
 SET(ILMBASE_VERSION_API ${ILMBASE_VERSION_MAJOR}_${ILMBASE_VERSION_MINOR})
@@ -160,31 +160,31 @@
 
 SET_TARGET_PROPERTIES ( Half
   PROPERTIES
-  VERSION 23.0.0
-  SOVERSION 23
+  VERSION 12.0.0
+  SOVERSION 12
   )
 SET_TARGET_PROPERTIES ( Iex
   PROPERTIES
-  VERSION 23.0.0
-  SOVERSION 23
+  VERSION 12.0.0
+  SOVERSION 12
   OUTPUT_NAME "Iex${ILMBASE_LIBSUFFIX}"
   )
 SET_TARGET_PROPERTIES ( Imath
   PROPERTIES
-  VERSION 23.0.0
-  SOVERSION 23
+  VERSION 12.0.0
+  SOVERSION 12
   OUTPUT_NAME "Imath${ILMBASE_LIBSUFFIX}"
   )
 SET_TARGET_PROPERTIES ( IlmThread
   PROPERTIES
-  VERSION 23.0.0
-  SOVERSION 23
+  VERSION 12.0.0
+  SOVERSION 12
   OUTPUT_NAME "IlmThread${ILMBASE_LIBSUFFIX}"
   )
 SET_TARGET_PROPERTIES ( IexMath
   PROPERTIES
-  VERSION 23.0.0
-  SOVERSION 23
+  VERSION 12.0.0
+  SOVERSION 12
   OUTPUT_NAME "IexMath${ILMBASE_LIBSUFFIX}"
   )
 
diff -urNad openexr-2.2.2-orig/IlmBase/configure.ac openexr-2.2.2/IlmBase/configure.ac
--- openexr-2.2.2-orig/IlmBase/configure.ac	2020-04-30 20:45:37.000000000 +0300
+++ openexr-2.2.2/IlmBase/configure.ac	2020-05-01 14:36:35.800035588 +0300
@@ -15,7 +15,7 @@
 AM_MAINTAINER_MODE
 
 
-LIBTOOL_CURRENT=23
+LIBTOOL_CURRENT=12
 LIBTOOL_REVISION=0
 LIBTOOL_AGE=0
 LIBTOOL_VERSION=$LIBTOOL_CURRENT:$LIBTOOL_REVISION:$LIBTOOL_AGE
